scipy/special/_support_alternative_backends.py

```diff
@@ -100,9 +119,25 @@ def __chdtrc(v, x):
     return __chdtrc
 
 
+def _betaincc(xp, spx):
+    betainc = getattr(spx, 'betainc', None)  # noqa: F811
+    if betainc is None and hasattr(xp, 'special'):
+        betainc = getattr(xp.special, 'betainc', None)
+    if betainc is None:
+        return None
+
+    def __betaincc(a, b, x):
+        # not perfect; might want to just rely on SciPy
+        return betainc(b, a, 1-x)
```

Comment:
@steppi This implements `betaincc` in terms of `betainc` for array backends that have `betainc` but not `betaincc`. I imagine that when `x` is close to `1`, `1 - betainc(a, b, x)` might be better for unusual values of `a` and `b`. I thought I'd check with you what to do here: is this good enough, is there something better we can do (possibly involving evaluating both versions and choosing the better one), or should we not provide this  sort of implementation?

Reply:
It's not going to work well or at all for small `x`, but I see no issues for now since the dispatch to alternative backends is only enabled when the experimental `SCIPY_ARRAY_API` flag is set. By the time what's now hidden behind `SCIPY_ARRAY_API=1` becomes the default behavior, we should have an implementation of `betaincc` that other backends can use. I don't think the values of `a` and ``b` for which a beta distribution's pdf has a sharp peak at `x=1` are that relevant for these NHSTs, so the time being this should work well enough for your use case on backends that don't support `betaincc`.

Reply:
Could this implementation and the naive 1-betainc be complementary? Like if the result is close to 1, then 1-betainc should be fine, right?

Reply:
Never mind I can imagine when the result is close to zero, and I suppose that tends to happen when x is close to 1.